LuaUI/widgets/chobby/components/chat_windows.lua

```diff
@@ -49,6 +49,12 @@ function ChatWindows:init()
 	lobby:AddListener("OnJoin",
 		function(listener, chanName)
 			local channelConsole = self:GetChannelConsole(chanName)
+			if channelConsole ~= nil then
+				WG.Delay(function()
+					channelConsole:AddMessage(lobby:GetChannel(chanName).topic, nil, nil, "\255\0\139\139")
+				end, 0.01)
+			end
+			
```

Comment:
Isn't it better to do all this in the OnChannelTopic callin?
Is there a guarantee this would be called after OnChannelTopic, or that it would work when the topic is updated?